pandas/core/series.py

```diff
@@ -5004,6 +5004,42 @@ def reindex(  # type: ignore[override]
             tolerance=tolerance,
         )
 
+    @overload
+    def rename_axis(  # type: ignore[override]
+        self,
+        mapper: ...,
```

Comment:
Please also add the `mapper` annotations to the overloads (`IndexLabel | lib.NoDefaul`)

Reply:
Done.